spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AppRegistryController.java

```diff
@@ -224,7 +223,7 @@ else if (entry.getKey().equals("outbound")) {
 	 * @param type module type
 	 * @param name module name
 	 * @param version module version
-	 * @param bootVersion module boot version or {@code null} to use the default
+	 * @param bootVersion module boot version or {@code null} to use the default.  Deprecated: bootVersion is no longer needed when registering an app.
```

Comment:
I have mixed feelings about leaving this in since there is no Boot2/3 in SCDF 3.x. However, since it is a controller it is the upgrade empathy thing to do to leave it in and mark `@Deprecated`. Do you mind adusting the desc to say that the parameter is ignored completely?